src/site/content/en/handbook/grammar/index.md

```diff
@@ -257,6 +258,18 @@ To copy, press `Control+C` (or `Command+C` on Mac).
 
 Use _pane_ to refer to the content areas associated with tabs—unless you're referring to an area in DevTools; then use _panel_.
 
+Use `about://` instead of `chrome://` for built-in URLs in web.dev articles. This generalizes these URLs to all Chromium forks. Typing `about://` in such a browser will redirect to a browser-specific page. For example typing `about://` in Microsoft Edge will redirect to `edge://`.
```

Comment:
```suggestion
Use `about://` instead of `chrome://` for built-in URLs in web.dev articles. This generalizes these URLs to all Chromium forks. Typing `about://` in such a browser will be rewritten to a browser-specific scheme. For example typing `about://` in Microsoft Edge will be rewritten to `edge://`.
```